packages/components/src/components/Table/Table.js

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import {
+  Button,
+  DataTable,
+  DataTableSkeleton,
+  TableBatchAction,
+  TableBatchActions,
+  TableSelectAll,
+  TableSelectRow,
+  TableToolbar,
+  TableToolbarContent
+} from 'carbon-components-react';
+import { ALL_NAMESPACES } from '@tektoncd/dashboard-utils';
+
+import './Table.scss';
+
+const {
+  TableContainer,
+  Table: CarbonTable,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableCell,
+  TableHeader
+} = DataTable;
+
+const Table = props => {
+  const {
+    batchActionButtons = [],
+    headers: dataHeaders,
+    loading,
+    rows: dataRows,
+    resource,
+    selectedNamespace,
+    sortable,
+    showTitle,
```

Comment:
change to `title`, this will be an optional prop that will be displayed if provided (safe to just pass it through to the TableContainer)